docs/testing/dotnet_testing_selenium.md

```diff
@@ -212,29 +212,6 @@ One of the best methods to test your locator is working is by trying it in the b
 
 [Data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) can help identify the elements under test.
 
-### Example
```

Comment:
I don't think this block should be deleted.
It references to Cypress, but it is actually for C# Selenium and therefore still relevant

Reply:
Section restored